pkg/appgateserver/synchronous.go

```diff
@@ -2,56 +2,60 @@ package appgateserver
 
 import (
 	"context"
-	"io"
 	"net/http"
 
 	sdktypes "github.com/pokt-network/shannon-sdk/types"
 
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
+// ruquestInfo is a struct that holds the information needed to handle a relay request.
```

Comment:
Why can't we reuse the `poktHTTPRequest` type here?

I understand this structure it convenient but I want to see if there's an opportunity to avoid having it.

Can you take a second look?

Reply:
`appAddress`, `serviceId` and `rpcType` are not part of `poktHTTPRequest`. The only alternative I can see is to have  each property as an argument passed to `handleSynchronousRelay` :disappointed: